modules/introductions/infrastructure/mongodb/demo-consent-attempt-repository-mongodb.ts

```
@@ -0,0 +1,180 @@
+			return docs.map(map);
+		},
+		async clear() {
+			try {
+				await collection.deleteMany({});
+			} catch (error) {
+				deps.logger.error({ event: "demo.consent.attempt.clear_failed", error: String(error) });
+				throw createDatabaseError("clear demo consent attempts", error);
+			}
+		},
+	};
+}
```

Biome (2.4.6)

[error] This function has too many lines (73). Maximum allowed is 60.

(lint/complexity/noExcessiveLinesPerFunction)